packages/twenty-front/src/modules/object-record/constants/BackendMergeRecordsMaxCount.ts

```diff
@@ -0,0 +1 @@
+export const BACKEND_MERGE_RECORDS_MAX_COUNT = 9;
```

Comment:
Use a single variable that is shared between backend and frontend, which you put in twenty-shared